solarized-definitions.el

```diff
@@ -192,8 +193,8 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
              (isearch ((t (,@fmt-stnd ,@fg-orange ,@bg-back)))) ; IncSearch
              (isearch-fail ((t (,@fmt-stnd ,@fg-orange ,@bg-back)))) ; IncSearch
              (lazy-highlight ((t (,@fmt-revr ,@fg-yellow ,@bg-back)))) ; Search
-             (link ((t (,@fmt-undr ,@fg-violet))))
-             (link-visited ((t (,@fmt-undr ,@fg-magenta))))
+             (link ((t (,@fmt-undr))))
+             (link-visited ((t (,@fmt-undr))))
```

Comment:
These have changed a bit since then, they now match what the Vim theme uses for `pandocLinkText`.